test/Microsoft.ML.StaticPipelineTesting/StaticPipeTests.cs

```diff
@@ -22,6 +22,7 @@
 using System.Text;
 using Xunit;
 using Xunit.Abstractions;
+using static Microsoft.ML.Transforms.Text.LdaTransform;
```

Comment:
Sort the usings.

Reply:
Control+R Control+G  (remove and sort Using) gives this order itself

---
In reply to: [231324047](https://github.com/dotnet/machinelearning/pull/1410#discussion_r231324047) [](ancestors = 231324047)